src/components/SectionTitle.vue

```diff
@@ -22,7 +22,7 @@
 <template>
 	<div id="priority">
 		<div class="app-content-list-item">
-			<h2>{{ name }}</h2>
+			<h3>{{ name }}</h3>
```

Comment:
@GretaD even though the size/look changes, semantically these headings have to stay h2.